docs/README.md

```diff
@@ -52,41 +54,39 @@ First, declare the `autocomplete-result-list.v2` block as a child block of the [
 | `productLayout`               | `enum`                  | Defines the suggested products list layout when rendered. Possible values are `HORIZONTAL` and `VERTICAL`.     | `undefined`   |
 | `hideTitles`                  | `boolean`               | Defines whether all component titles are hidden when rendered (`true`) or not (`false`).              | `false`       |
 | `hideUnavailableItems`                  | `boolean`               | Defines whether the autocomplete should hide unavailable items (`true`) or not (`false`).             | `false`       |
 | `historyFirst`                | `boolean`               | Defines whether the search history list should be prioritized over the other lists (`true`) or not (`false`).                                                          | `false`       |
 | `customBreakpoints`           | `object`                | Defines a maximum number of suggested products by breakpoints. Possible values are `md`, `lg` or `xlg`.                                                           | -             |
-| `simulationBehavior`          | `"skip"` or `"default"` | If you want faster searches and do not care about most up to date prices and promotions, use `"skip"` value.                                                                                                                                                                                                                                                                                                                                                                                    | `default`     |
-| `HorizontalProductSummary`          | `product-summary` block | By default, the mobile autocomplete uses the `CustomListItem` component to render the suggested products with a horizontal layout. But if you send a `product-summary` block here, it will render your customized Product Summary component. You can see how to build a horizontal Product Summary component [here](https://vtex.io/docs/recipes/templates/building-a-horizontal-product-summary/)                            | `undefined`     |
+| `simulationBehavior`          | `"skip"` or `"default"` | If you want faster searches and do not care about most up to date prices and promotions, use `"skip"` value.     | `default`     |
+| `HorizontalProductSummary`          | `product-summary` block | By default, the mobile autocomplete uses the `CustomListItem` component to render the suggested products with a horizontal layout. But if you send a `product-summary` block here, it will render your customized Product Summary component. Read our documentatio of [how to build a horizontal Product Summary](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-building-a-horizontal-product-summary) component.                            | `undefined`     |
```

Comment:
```suggestion
| `HorizontalProductSummary`          | `product-summary` block | By default, the mobile autocomplete uses the `CustomListItem` component to render the suggested products with a horizontal layout. But if you send a `product-summary` block here, it will render your customized Product Summary component. Read our documentation of [how to build a horizontal Product Summary](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-building-a-horizontal-product-summary) component.                            | `undefined`     |
```